pkg/bosh/converter/volume_factory.go

```diff
@@ -496,3 +499,22 @@ func generateVolumeName(secretName string) string {
 	}
 	return names.Sanitize(volName)
 }
+
+func linkVolume(secretName string) corev1.Volume {
```

Comment:
I find the name to generic. If these methods were on a struct, this could read `quarksLinks.volume()` or something.